lib/modules/apostrophe-ui/public/css/components/tooltip.less

```diff
@@ -0,0 +1,35 @@
+.apos-tooltip
+{
```

Comment:
right now its being imported at the bottom of `apostrophe-ui/public/css/user.less`